### ISSUE TITLE
Ibis server error

### ISSUE THREAD

**Issue by m7mdhka** [Canner/WrenAI#652]
**Describe the bug**
![image](https://github.com/user-attachments/assets/edc8d1af-2c50-418c-9f5f-c58e6a2d6855)

**To Reproduce**
1. Go to Home
2. Click on Ask
3. See error

**Desktop (please complete the following information):**
- Browser chrome

**Wren AI Information**
- LLM_PROVIDER= azure_openai_llm
- GENERATION_MODEL= # gpt-4o-mini

**Generated SQL**
```
WITH
  filtered_customers AS
  -- Selects customer IDs and their default names from the MiAutoCare_Customer table, ensuring that customer IDs are not null.
  (
    SELECT
      "CustomerID",
      "CustomerDefaultName"
    FROM
      "MiAutoCare_Customer" AS "Customer"
    WHERE
      NOT "CustomerID" IS NULL
  )
  -- Limits the results to the top 5 customers based on the filtered data.
SELECT
  "CustomerID",
  "CustomerDefaultName"
FROM
  "filtered_customers"
LIMIT
  5
  ```

**Comment by m7mdhka** [comment]
[Logs.txt](https://github.com/user-attachments/files/16893243/Logs.txt)

**Comment by cyyeh** [comment]
@grieve54706 could you take a loot at this issue? thank you

**Comment by grieve54706** [comment]
Sure.

**Comment by grieve54706** [comment]
@cyyeh, @wwwy3y3
It's fixed at [wren-engine@0.10.0](https://github.com/Canner/wren-engine/releases/tag/0.10.0).
I'm not sure this version will be merged into which version of WrenAI. Please help and close this issue.
